locationsharinglib/locationsharinglib.py

```diff
@@ -158,24 +158,24 @@ def _get_data(self):
                           '1f1.3953487873077393!39b1!44e1!50e0!23i4111425')}
         url = 'https://www.google.com/maps/rpc/locationsharing/read'
         response = self._session.get(url, params=payload, verify=True)
-        self._logger.debug(response.text)
+        self._logger.debug('Response status: %s, body: %s', response.reason, repr(response.text))
         if response.ok:
             try:
                 data = json.loads(response.text.split("'", 1)[1])
-            except (ValueError, IndexError, TypeError):
-                self._logger.exception('Unable to parse response :%s',
-                                       response.text)
+            except (ValueError, IndexError, TypeError) as e:
```

Comment:
Here you don't need the ```as e``` since we log exception level so the exception stack will be printed. So the str(e) is also not required.

Reply:
Thanks, I've fixed that.

Reply:
Thanks! Sorry for missing it, but you also do not need ```repr(response.text)``` since the ```.text``` variable is already a string so there is no need to cast with repr.

Reply:
OK, corrected.